src/utils/work/getExtraDependencies.js

```diff
@@ -36,6 +41,19 @@ const getCellSets = async (experimentId, body, dispatch, getState) => {
   return hierarchy;
 };
 
+const getGroupedTracksCellSets = async (experimentId, body, dispatch, getState) => {
```

Comment:
this function looks like a copy of `getSelectedCellSet`. Why do we need it?

Reply:
it takes `groupedTracks` instead of `selectedCellSet`. Although I guess that I could calculate both together since they are dependencies that always go together, let me know if that makes sense to you

Reply:
If you think that we will need to over-generalise too much, then in my opinion the code will be clearer to read if we keep them as separate functions. I notice however that both `getGroupedTracksCellSets` and `getSelectedCellSet` are used in the same places, namely in `GeneExpression` and `MarkerHeatmap`. I wonder if it is worth having only 1 function that returns both?

A small thing if you want to keep them as separate functions:
naming of "getSelectedCellSet" vs "getGroupedTracksCellSet`s`" - does "getSelectedCellSet" return a single cell set? If not, it makes sense to add an `s` similarly to "getGroupedTracksCellSets".

Reply:
Good point. I will look into lumping them together

Reply:
Done, let me know what you think

Reply:
looks good!